### PR TITLE
Move over to new Azure SDK

### PR DESCRIPTION
**Pull request by tzhanl**
**Purpose**
---------------------------------------------------------------------------------------------------------------
Move over this sample to new Azure SDK:
 1. Move the original file to storage-blob-upload-from-webapp-node-v10-v3 folder
 2. Add folder storage-blob-upload-from-webapp-node-v10-v4 for the sample referenced to the new SDK
 3. Add the readme to introduce the Sample and folders in the root folder.

**What old sample do** 
---------------------------------------------------------------------------------------------------------------
image

**What can be updated to V4:**
---------------------------------------------------------------------------------------------------------------
image

### CODE REVIEW

File: storage-blob-upload-from-webapp-node-v10-v4/views/index.hbs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<h1>Image Resizer</h1>
+
+<h2>Upload photos</h2>
```
Comment (jongio) [inline storage-blob-upload-from-webapp-node-v10-v4/views/index.hbs]: Case: Photos

File: storage-blob-upload-from-webapp-node-v10-v4/config.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+const _module = {
+    getStorageAccountName: () => {
```
[the remark this answers is not in the document]
Comment (tzhanl) [reply storage-blob-upload-from-webapp-node-v10-v4/config.js]: not need config.js, removed

File: storage-blob-upload-from-webapp-node-v10-v4/routes/upload.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+if (process.env.NODE_ENV !== 'production') {
```
[the remark this answers is not in the document]
Comment (tzhanl) [reply storage-blob-upload-from-webapp-node-v10-v4/routes/upload.js]: removed upload.js

File: README.md [hunk 4]
```diff
@@ -1,29 +1,23 @@
----
-page_type: sample
```
Comment (jongio) [inline README.md]: Why are you removing the front-matter?

File: README.md [hunk 5]
```diff
@@ -1,29 +1,23 @@
...
-- azure-storage
-description: "This sample demonstrates how to use the Azure Storage v10 SDK in the context of an Express application to upload images into Azure Blob Storage."
----
-
 # Azure Storage Blob Upload from a Node.js Web Application using the v10 SDK
 
 This sample demonstrates how to use the Azure Storage v10 SDK in the context of an [Express](https://expressjs.com/) application to upload images into Azure Blob Storage.
 
-## Getting started
+## SDK Versions
```
Comment (jongio) [inline README.md]: See comment on key-vault python example and use same format here.

File: README.md [hunk 6]
```diff
@@ -32,6 +26,14 @@ Install dependencies via `npm`:
 npm install
 ```
 
+## In this sample you will do the following:
+
+* Create a storage account.
+* Create a container.
+* Upload a stream to [blockblob](https://docs.microsoft.com/en-us/rest/api/storageservices/understanding-block-blobs--append-blobs--and-page-blobs).
+
+If you don't have a Microsoft Azure subscription, you can get a free trial account <a href="http://go.microsoft.com/fwlink/?LinkId=330212">here</a>.
```
Comment (jongio) [inline README.md]: This trial account text should be in the same location as all the other samples.

File: storage-blob-upload-from-webapp-node-v12/.env.example [hunk 7]
```diff
@@ -0,0 +1,2 @@
+AZURE_STORAGE_ACCOUNT_NAME=<replace with your storage account name>
```
Comment (jongio) [inline storage-blob-upload-from-webapp-node-v12/.env.example]: You consistent placeholders `<Pascal>`

**Pull request merged**